utils/clients/tools/tests/test_isisicat_prefix_mapping.py

```diff
@@ -0,0 +1,87 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Test ICAT prefix mapping
+"""
+import unittest
+from unittest.mock import patch
+
+from utils.clients.tools.isisicat_prefix_mapping import fetch_instrument_fullname_mapping
+
+
+# pylint:disable=no-self-use,too-few-public-methods,too-many-public-methods
+class MockInstrumentQueryResult:
+    """
+    Mocks result of isisicat_prefix_mapping.client.execute_query for an instrument
+    """
+
+    def __init__(self, name, full_name):
+        self.name = name
+        self.fullName = full_name  # pylint:disable=invalid-name
+
+
+class TestICATPrefixMapping(unittest.TestCase):
+    """
+    Test ICAT prefix mapping
+    """
+
+    @patch('icat.Client.__init__', return_value=None)
+    @patch('utils.clients.tools.isisicat_prefix_mapping.AUTOREDUCTION_INSTRUMENT_NAMES',
```

Comment:
To make some of the patches look a bit neater/fit on one line, may be worth placing regularly referenced paths in a variable such as:
`DIR = "utils.clients.tools.isisicat_prefix_mapping"`
and then reference when needed such as:
`@patch(f"{DIR}.AUTOREDUCTION_INSTRUMENT_NAMES", return_value = ['ENGINGX']`)